solutions/aiven-iceberg-tutorial/terraform/aws_setup/terraform.tfvars.example

```diff
@@ -0,0 +1,5 @@
+aws_region     = "us-west-2"
+aws_account_id = "123456789012"
```

Comment:
@Douglas-Young - Let's make sure we don't have any existing account ids but just a placeholder such as "your_aws_account_id" wherever applicable